lib/bitgo_client/v2.rb

```diff
@@ -80,6 +72,15 @@ def wallet(wallet_id, coin_code: :tbtc, logger: nil, all_tokens: nil)
       client.request("#{base_path}/#{coin_code}/wallet/#{wallet_id}?#{query_string}", logger: logger)
     end
 
+    def send_transaction(wallet_id, payload, coin_code: :tbtc, logger: nil)
+      client.request(
+        "#{express_path}/api/v2/#{coin_code}/wallet/#{wallet_id}/sendcoins",
+        payload,
+        method: :post,
+        logger: logger
+      )
+    end
+
```

Comment:
I think we should keep the change small so it's easier for upstream to accept. Moving methods should happen for good reason and in a separate PR

Reply:
Ah yes, I should have done this in a separate commit or something. The reason I moved this is so we can group the methods that uses express and those which do not.